pkg/standalone/standalone.go

```diff
@@ -55,6 +56,31 @@ func (h *delegateHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	w.WriteHeader(http.StatusNotFound)
 }
 
+// TODO: replace this with clientcmd
+func GetAPIServerClient(authPath string, apiServerList util.StringList) (*client.Client, error) {
```

Comment:
Are you referring to the variable?  IIRC all variables start lowercase, even as initialisms (httpServer, url).  Or something else?


> On Jan 7, 2015, at 6:38 PM, Eric Tune <notifications@github.com> wrote:
> 
> In pkg/standalone/standalone.go:
> 
> > @@ -55,6 +56,31 @@ func (h *delegateHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
> >  	w.WriteHeader(http.StatusNotFound)
> >  }
> >  
> > +// TODO: replace this with clientcmd
> > +func GetAPIServerClient(authPath string, apiServerList util.StringList) (*client.Client, error) {
> https://github.com/golang/go/wiki/CodeReviewComments#initialisms
> 
> —
> Reply to this email directly or view it on GitHub.
>